jetty-osgi/test-jetty-osgi/pom.xml

```diff
@@ -105,6 +118,12 @@
     </dependency>
 
     <!-- Jetty OSGi Deps -->
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-slf4j-impl</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
```

Comment:
FYI, this dependency (`jetty-slf4j-impl`) is declared at the top level `pom.xml` with both a version and scope.
_(This is just a comment, no change is being asked for)_